pkg/storegateway/series_refs.go

```diff
@@ -955,7 +955,7 @@ func (s *loadingSeriesChunkRefsSetIterator) Next() bool {
 	}
 	defer symbolizedSet.release() // We only retain the slices of chunk ranges from this set. These are still not pooled, and it's ok to retain them.
 
-	nextSet, err := s.stringifiedSet(symbolizedSet)
+	nextSet, err := s.stringifiedSet(context.TODO(), symbolizedSet)
```

Comment:
Adding a context argument to the `Next` method causes a domino effect of other iterator types that'll need the same change, so I opted for `context.TODO()` for now. I'm afraid the change set will be too big otherwise.

Reply:
Could you use `s.ctx` ?

Reply:
Switched to `s.ctx`.